src/sentry/scim/endpoints/teams.py

```diff
@@ -186,17 +180,48 @@ def post(self, request: Request, organization) -> Response:
                 "idp_provisioned": True,
             }
         )
-        metrics.incr("sentry.scim.team.provision", tags={"organization": organization})
-        return super().post(request, organization)
+        metrics.incr("sentry.scim.team.provision")
+        serializer = TeamPostSerializer(data=request.data)
+
+        if serializer.is_valid():
+            result = serializer.validated_data
+
+            team = organization_service.create_team(
+                organization_id=organization.id,
+                slug=result["slug"],
+                name=result.get("name"),
+                idp_provisioned=result.get("idp_provisioned", False),
+            )
```

Comment:
If scim is region bound now, do we still need an service call to make teams?

Reply:
Yup, this was inlined and removed.